great_expectations/cli/datasource.py

```diff
@@ -370,9 +371,16 @@ def _add_pandas_datasource(
             }
         )
 
-    context.add_datasource(
-        name=datasource_name, class_name="PandasDatasource", **configuration
+    cli_message("""
+Great Expectations will now add a new Datasource '{0:s}' to your deployment, by adding this entry to your great_expectations.yml:
+
+{1:s}
+""".format(datasource_name, textwrap.indent(toolkit.yaml.dump({datasource_name: configuration}), "  "))
```

Comment:
The `configuration` object here should have the appropriate `class_name` added to it to ensure that what's displayed is *exactly* what will be added to the user's config.

I think this should also be validated using DatasourceConfigSchema:
```
configuration["class_name"] = "PandasDatasource"
errors = DatasourceConfigSchema().validate(configuration)
if len(errors) != 0:
  # handle errors
```

Reply:
Resolved.